getting_started.md

```diff
@@ -0,0 +1,109 @@
+# Getting Started
+
+## Workspace set up and installation
+To get started you will have to make a few decisions and install the repository along with its dependencies. Specifically:
+1. Decide if you would like to develop your submission in either Pytorch or Jax.
+2. Set up your workstation or VM. We recommend to use a setup similar to the competition scoring hardware which consists out of 8 V100 GPUs, 350 GB in RAM and 2 TB in storag for dataset storage. For recommendations on setting up your own Cloud VM see [here](https://github.com/mlcommons/algorithmic-efficiency/blob/main/docker/README.md#gcp-integration).
+4. Clone the algorithmic-efficiency repository and make sure you have installed the dependencies. To install the dependencies we recommend either:
+    1. Installing in a python virtual environment as described in the [Installation section in the README](https://github.com/mlcommons/algorithmic-efficiency/blob/main/README.md) instructions.
+    2. Use a Docker container with dependencies installed. This option will guarantee that you're code is running on the same CUDA, cudnn, and python dependencies as the competition scoring environment. See the [Docker README](https://github.com/mlcommons/algorithmic-efficiency/blob/main/docker/README.md) for instructions on the Docker workflow.
+
+## Download the data
+The workloads in this benchmark use 6 different datasets across 8 workloads. You may choose to download some or all of the datasets as you are developing your submission, but your submission will be scored across all 8 workloads. For instructions on obtaining and setting up the datasets see [datasets/README](https://github.com/mlcommons/algorithmic-efficiency/blob/main/datasets/README.md#dataset-setup).
+
+
+## Develop your submission
+To develop a submission you will write a python module containing your optimizer algorithm. Your optimizer must implement a set of predefined API methods for the initialization and update steps.
+
+### Set up your directory structure (Optional)
+
+1. Make a submissions subdirectory to store your submission modules e.g. `algorithmic-effiency/my_submissions`.
+2. Make an experiment directory to store your experiment results e.g. `~/my_experiments`.
+
+### Coding your submission
+#TODO(kasimbeg): If we want use this submission_template script where should we store it in the repo?
+
+You can find examples of sumbission modules under `algorithmic-efficiency/baselines` and `algorithmic-efficiency/reference_algorithms`. \
+A submission for the external ruleset will consist of a submission module and a tuning search space definition.
+1. Clone the submission_template.py script into your submissions directory e.g. in `algorithmic-efficiency/my_submissions`.
+2. Implement at least the methods in the submission_template. Feel free to use helper functions and/or modules as you see fit.
```

Comment:
can you link to https://github.com/mlcommons/algorithmic-efficiency/blob/main/RULES.md#software-dependencies